avm/res/maintenance/configuration-assignment/tests/e2e/winvm.multirg/main.test.bicep

```diff
@@ -0,0 +1,74 @@
+targetScope = 'subscription'
+
+metadata name = 'Multi resource group'
+metadata description = 'This instance deploys the module leveraging virtual machine and maintenance configuration dependencies from two different resource groups. This instance assigns an existing Windows virtual machine to the input maintenance configuration.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-maintenance.maintenanceconfigurations-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'mcamrg'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+#disable-next-line no-hardcoded-location
+var enforcedLocation = 'uaenorth'
+
+// General resources
+// =================
+resource resourceGroup_vm 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: enforcedLocation
+}
+
+resource resourceGroup_mc 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: '${take(resourceGroupName, 87)}-mc' // Ensure the resource group name is within the 90 character limit
+  location: enforcedLocation
+}
+
+module nestedDependencies_vm 'dependencies_vm.bicep' = {
+  scope: resourceGroup_vm
+  name: '${uniqueString(deployment().name, enforcedLocation)}-nestedDependencies_vm'
+  params: {
+    virtualMachineName: 'dep-${namePrefix}-vm-${serviceShort}'
+    computerName: 'dep${namePrefix}${serviceShort}'
+    adminUsername: 'localAdminUser'
```

Comment:
Same comment as in waf test

Reply:
Same reply 😄

Reply:
Still find it odd. Maybe a default in the dependencies.bicep file would be an alternative that is more aligned. Anyways, will resolve